src/sigchain/sigchain.iced

```diff
@@ -200,6 +214,13 @@ class ChainLink
     await sig_eng.get_body_and_unverified_payload(
       {armored: sig_blob.sig}, esc defer sig_body, unverified_buffer)
     sig_id = kbpgp.hash.SHA256(sig_body).toString("hex") + SIG_ID_SUFFIX
+
+    # This is the earliest possible point we can identify known buggy sigs,
+    # right after we've computed the SigID. Callers should catch this error.
```

Comment:
I think it's also fine to collect payload hashes from the buggy chainlinks. This would prevent the workaround below.